### DISCUSSION TITLE
Overview with ALL tasks assigned to me

### DISCUSSION THREAD

**Discussion by S4-Mize** [Leantime/leantime#989] [Q&A]
hey there,

we are a team of 4 and working on different projects with leantime. 
Now I would like to know, if there is any possibilty to see all my tasks (witch are assigned to me no matter in what project) in an overview? 
At the moment I only see my ToDos in the overview of one project. If I have a new task in an other project, I only see it via mail notification. 
Does anyone have an idea, how this could be possible to see all my tasks at one glance? 
Thanks

**Comment by gloriafolaron** [comment]
Hi!

Great idea.

This feature went in with version 2.2.8. We're on 2.2.9 right now.

Highly recommend updating if you haven't yet or if you wait another day or two, we'll be releasing some more updates.

Thanks for the suggestions!

**Comment by kfklein15** [comment]
Have you been able to figure out how to do this? The answer you got didn't explain HOW to do it, just that it is already available.

**Comment by marcelfolaron** [comment]
The overview is on your "Dashboard" (the home button on the top menu of the
screen).
All of those tasks are tasks assigned to you across all projects.

**Reply by marcelfolaron** [reply]
@S4-Mize were you able to find the screen?
